build.gradle

```diff
@@ -44,9 +44,12 @@ repositories {
 dependencies {
 	compile "com.google.code.gson:gson:2.2.2"
 	compile "org.java-websocket:Java-WebSocket:1.4.0"
+
 	testCompile "org.mockito:mockito-all:1.8.5"
 	testCompile "org.powermock:powermock-module-junit4:1.4.11"
 	testCompile "org.powermock:powermock-api-mockito:1.4.11"
+
+	testImplementation "com.google.truth:truth:1.0.1"
```

Comment:
Not sure why my commits made it here and even why are they displaying in the GH diff as that's already merged (it's not displaying when I compare the branches in IntelliJ), but maybe we could rebase interactively the branch dropping my commits? Alternatively we need to remember to squash merge to keep the code history clean.

Reply:
I believe it's because of the use of merge, rather than re-basing the branch.

The github diff view shows the diffs based on the changes applied in this history vs when it branched from the base, which is useful (in that it doesn't show changes in the base branch that happened while you were working) but is easily confused by merges from the base branch in to dev branches.

But your commit introducing this line only exists once in the entire history, in d851213.

Reply:
I merged your branch in before it made it to the integration branch yesterday. I'll rebase now.

Reply:
This issue has been fixed with merging in the integration branch - there's now only 8 files changed 🙃